CHANGELOG.md

```diff
@@ -7,6 +7,10 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
+## Unrealeased
+* Add page heading captions to checkboxes component ([PR #1304](https://github.com/alphagov/govuk_publishing_components/pull/1304))
+* Add page heading captions to radio boxes component ([PR #1304](https://github.com/alphagov/govuk_publishing_components/pull/1304))
```

Comment:
You don't need to add two entries for this, it's one PR.

Reply:
Ah ha, great I'll change this up